### ISSUE TITLE
Where can I find a description of the "features"?

### ISSUE THREAD

**Issue by hexists** [MeetKai/functionary#103]
Thank you for the project.
Where can I find a description of the "features"?
https://github.com/MeetKai/functionary?tab=readme-ov-file#related-projects--their-differences

**Comment by hexists** [comment]
Thank you for your quick and detailed response. Have a good day.

**Comment by hexists** [comment]
@khai-meetkai 

I have one more question.
In your opinion, does "code interpreter" ability count as function calling ability?
By "code interpreter" do you mean the ability to write actual code?

**Comment by khai-meetkai** [comment]
@hexists 
yes code interpreter can be considered as function calling, and the arguments is the python code.
For example, 
Query="what is the result of 199+100+345"
--> python("a=199+100+345\nprint(a)")
So Code interpreter is the ability to write code to handle the query from User.
Our current models (Version 2.2) **do support code interpreter** but the quality is **not good enough** so we decided to **remove** this feature from our models. We will include Code Interpreter to **our next Version 2.3**

**Comment by hexists** [comment]
@khai-meetkai 
Thanks to your explanation, I fully understand the features of the model.
I really like the project and appreciate your contribution.
